std/bigint.d

```diff
@@ -1761,3 +1761,111 @@ unittest
     auto b = immutable BigInt("123");
     assert(b == 123);
 }
+
+/// Extracts the quotient and remainder for the given dividend and divisor.
+void divMod(const BigInt dividend, const BigInt divisor, out BigInt quotient, out BigInt remainder) pure nothrow
+{
+    BigUint q, r;
+    BigUint.divMod(dividend.data, divisor.data, q, r);
+    quotient.sign = dividend.sign != divisor.sign;
+    quotient.data = q;
+    remainder.sign = dividend.sign;
+    remainder.data = r;
+}
+
+/// ditto
+void divMod(T)(const BigInt dividend, T divisor, out BigInt quotient, out BigInt remainder) pure nothrow
+    if (is(typeof(BigInt(divisor))))
+{
+    divMod(dividend, BigInt(divisor), quotient, remainder);
+}
+
+/// ditto
+void divMod(T)(T dividend, const BigInt divisor, out BigInt quotient, out BigInt remainder) pure nothrow
+    if (is(typeof(BigInt(dividend))))
+{
+    divMod(BigInt(dividend), divisor, quotient, remainder);
+}
```

Comment:
There's not a lot of value in these last two convenience functions - after all, client code may simply construct the BigInt themselves. Worse, their existence suggest the caller some good optimizations are at work when in fact none is. Let's drop the overloads and only keep the first.